acceptance/tests/resource/service/should_query_all.rb

```diff
@@ -0,0 +1,9 @@
+test_name "should query all services"
+
+agents.each do |agent|
+  step "query with puppet"
+  on(agent, puppet_resource('service'), :accept_all_exit_codes => true) do
+    assert_equal(exit_code, 0, "'puppet resource service' should have an exit code of 0")
+    assert(/^service/ =~ stdout, "'puppet resource service' should present service details")
```

Comment:
@MosesMendoza I tested, this will match any line in the output. So it's asserting that we have at least one line starting with "service"